localstack/services/events/events_starter.py

```diff
@@ -56,6 +56,20 @@ def send_event_to_firehose(event, arn):
         Record={'Data': to_bytes(json.dumps(event))})
 
 
+def send_event_to_event_bus(event, arn):
+    bus_name = arn.split(':')[-1].split('/')[-1]
+    events_client = aws_stack.connect_to_service('events')
+    print(event, event.get('source'), event.get('detail-type'), event.get('detail'))
```

Comment:
Looks great, thanks for this PR @usmangani1 👍  - can we please remove this `print` before merging? Thanks